infrastructure/main.tf

```diff
@@ -57,33 +57,33 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
 }
 
 resource "azurerm_key_vault_secret" "POSTGRES-USER-V11" {
-  name         = "recipe-backend-POSTGRES-USER"
+  name         = "recipe-backend-POSTGRES-USER-v11"
   value        = module.recipe-database-v11.user_name
-  key_vault_id = data.azurerm_key_vault.key_vault.id
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}"
 }
 
 resource "azurerm_key_vault_secret" "POSTGRES-PASS-V11" {
-  name         = "recipe-backend-POSTGRES-PASS"
+  name         = "recipe-backend-POSTGRES-PASS-v11"
   value        = module.recipe-database-v11.postgresql_password
-  key_vault_id = data.azurerm_key_vault.key_vault.id
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}"
 }
 
 resource "azurerm_key_vault_secret" "POSTGRES_HOST-V11" {
-  name         = "recipe-backend-POSTGRES-HOST"
+  name         = "recipe-backend-POSTGRES-HOST-v11"
   value        = module.recipe-database-v11.host_name
-  key_vault_id = data.azurerm_key_vault.key_vault.id
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}"
```

Comment:
don't add interpolation syntax when not needed this triggers deprecation warnings in terraform, there's no need to change this or any of the other key vault ids